tests/aws/services/secretsmanager/test_secretsmanager.snapshot.json

```diff
@@ -3788,19 +3786,18 @@
         },
         "VersionIdsToStages": {
           "<version_uuid:1>": [
-            "AWSPREVIOUS"
+            "AWSCURRENT"
```

Comment:
q: do you understand why this changed?

Reply:
My assumption for this. The list is not deterministic as I noticed that sometimes version1 could be the previous version and version 2 is the current version and vice versa. Thats the reason why sometimes version 1 could be previous and sometimes it could be current. The same thing for version 2 as well